storage/LegacyStorageQuickstart/Podfile

```diff
@@ -1,3 +1,5 @@
+source 'https://github.com/firebase/SpecsDev.git'
```

Comment:
nit: maybe leave a comment here that this should be removed after V9 launch

Reply:
done.